hackathon-website/src/Components/SideDrawer/SideDrawer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./SideDrawer.css";
+import { NavbarData } from "../Data/NavbarData";
+import NavbarItems from "../Items/NavbarItems";
+
+const SideDrawer = (props) => {
+  let drawerClasses = "side_drawer";
+  if (props.show) {
+    drawerClasses = "side_drawer open";
+  }
+  return (
+    <nav className={drawerClasses}>
+      <ul>
+        <NavbarItems data={NavbarData} iconite={1} />
```

Comment:
Keep overall language consistency through the code.
Orice nume de variabila sa fie in engleza pe peste tot in cod.

Schimba din "iconite" in "icons".